packages/guui/components/CloseButton/CloseButton.stories.tsx

```diff
@@ -6,7 +6,14 @@ import { Appearances } from '@guardian/pasteup/palette';
 
 type contrastKeys = keyof Appearances['contrasts'];
 
-const stories = storiesOf('User Interface', module).addDecorator(withKnobs);
+const stories = storiesOf('User Interface', module)
+    .addDecorator(withKnobs)
+    .addParameters({
+        backgrounds: [
+            { name: 'Dark background', value: '#121212', default: true },
```

Comment:
can we import the colors from `pasteup`?

Reply:
yes